x/sequencers/types/permission.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (ap *AddressPermissions) Validate() error {
+	if strings.TrimSpace(ap.Address) == "" {
+		return errors.New("address field cannot be blank")
+	}
+	if _, err := sdk.AccAddressFromBech32(ap.Address); err != nil {
+		return fmt.Errorf("address format error: %s", err.Error())
+	}
+
+	return ap.Permissions.Validate()
+}
+
+func (p *Permissions) Validate() error {
+	if len(p.Permissions) == 0 {
+		return errors.New("permissions field cannot be empty")
+	}
```

Comment:
why?

Reply:
extra branch for no reason I think

Reply:
Why do you think it is not needed? In this case, we can get rid of the empty proposal that happen in our chain, cause even when check for empty permissions

Reply:
it's just an extra branch for no reason
let the empty proposal happen